engine/physics/src/physics/test/test_physics.cpp

```diff
@@ -90,6 +90,10 @@ bool ContactPointCallback(const dmPhysics::ContactPoint& contact_point, void* us
     }
 }
 
+#if defined(PHYSICS_TEST_BULLET_3D)
```

Comment:
Perhaps a bit icky, but I didn't want to completely restructure these tests. We now create three test exes, one for each physics implementation that can run the same test functions but with different libraries.